llvm/lib/Target/RISCV/RISCVSchedSpacemitX60.td

```diff
@@ -232,9 +392,660 @@ let Latency = 4 in {
   def : WriteRes<WriteFMovI32ToF32, [SMX60_IEU]>;
 }
 
+// 6. Configuration-Setting Instructions
+def : WriteRes<WriteVSETVLI, [SMX60_IEUA]>;
+def : WriteRes<WriteVSETIVLI, [SMX60_IEUA]>;
+def : WriteRes<WriteVSETVL, [SMX60_IEUA]>;
+
+// 7. Vector Loads and Stores
+// TODO: These latencies are estimations and are not confirmed experimentally
+foreach mx = SchedMxList in {
+  defvar IsWorstCase = SMX60IsWorstCaseMX<mx, SchedMxList>.c;
+
+  // Unit-stride loads and stores
+  let Latency = Get44816Latency<mx>.c in {
+    defm "" : LMULWriteResMX<"WriteVLDE", [SMX60_VLS], mx, IsWorstCase>;
+    defm "" : LMULWriteResMX<"WriteVLDFF", [SMX60_VLS], mx, IsWorstCase>;
+    defm "" : LMULWriteResMX<"WriteVSTE", [SMX60_VLS], mx, IsWorstCase>;
+  }
+
+  // Mask loads and stores
+  let Latency = 3 in {
+    defm "" : LMULWriteResMX<"WriteVLDM", [SMX60_VLS], mx, IsWorstCase=!eq(mx, "M1")>;
+    defm "" : LMULWriteResMX<"WriteVSTM", [SMX60_VLS], mx, IsWorstCase=!eq(mx, "M1")>;
+  }
+
+  // Strided and indexed loads and stores: scale with both LMUL and EEW
+  foreach eew = [8, 16, 32, 64] in {
+    defvar EEWMultiplier = !div(eew, 8);
```

Comment:
Hm, this seems backwards from what I expect with larger EEWs being more expensive?  I would expect this to scale with the number of elements, and thus have smaller EEWs be more expensive.